src/bp/core/app/bootstrap.ts

```diff
@@ -106,6 +106,22 @@ async function prepareLocalModules(app: BotpressApp, logger: sdk.Logger) {
   await app.ghost.root().syncDatabaseFilesToDisk('modules')
 }
 
+const runStan = (): Promise<{ code: number | null; signal: string | null }> => {
+  const command = process.env.STAN_DEV_MODE ? 'yarn start nlu' : './stan'
```

Comment:
:+1:

Reply:
will this work on a windows binary ?

Reply:
will have to check if we need to detect the OS and add a `.exe` will depend on the binary build process